docs/index.asciidoc

```diff
@@ -47,10 +47,9 @@ within 30 days of a database update.
 
 The GeoIP filter plugin can manage the database for users running the Logstash default
 distribution, or you can manage
-database updates on your own. The behavior is controlled by the `database` setting.
-When you use the default `database` setting, the auto-update feature ensures that the plugin is
+the updates on your own, becoming also responsible for maintaining compliance. The behavior is controlled by the `database` setting.
```

Comment:
I'd like to see the note about responsibility moved back to the end of the paragraph.  People tend to read headings, skim through the middle, and read the last sentence. I'm concerned that this important information about responsibility shifting to the user will be overlooked in its current location.

Reply:
Thank you for the clarification, that's nice to know! I can maybe reword line 50 a bit to include the auto-update and move everything back to the end of the paragraph as you suggested.

```
database updates on your own. The behavior is controlled by the `database` setting and by the auto-update feature.
```

Reply:
Sounds good.